### PR TITLE
[F] Update preisatlas profiles

### PR DESCRIPTION
**Pull request by SinisaG**
Added preisatlas test config. 
Additionally updated urls to respond to latest structure and use same protocol and test profiles, to make sure comparison is relevant. 

@viniciuskneves: This is how you can add profiles. 
The intervals are set in the profile, but you can also trigger the test manually, through this: https://speedtracker.org/test

### CODE REVIEW

File: _profiles/preisatlas-test.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+interval: 12
+
+#
+# WebPageTest test parameters.
+# For a list of supported params, check:
+#
+# https://www.npmjs.com/package/webpagetest
+#
+parameters:
+  connectivity: "Cable"
```
Comment (SinisaG) [inline _profiles/preisatlas-test.html:28]: You can change the connection and devices with config here.

File: _profiles/preisatlas-test.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+# (in milliseconds), and one or multiple alerts — referenced by the ids defined
+# in the main config — to be triggered when a budget is overrun.
+#
+# For a list of metrics, check:
+#
+# https://github.com/speedtracker/speedtracker-api/blob/master/lib/SpeedTracker.js#L18-L37
+#
+#budgets:
+#  -
+#    metric: TTFB
```
Comment (SinisaG) [inline _profiles/preisatlas-test.html:53]: And setup different metrics and budgets

File: _profiles/preisatlas-test.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+#
+# The human-friendly name of the profile. Whenever you're asked for the profile
+# ID, it refers to the name of the profile file — in this case, "default".
+#
+name: "Preisatlas Staging"
```
Comment (viniciuskneves) [inline _profiles/preisatlas-test.html]: @SinisaG : rename

Comment (SinisaG) [reply _profiles/preisatlas-test.html]: Fixed. ;)

**Pull request merged**